packages/assert/src/assert.js

```diff
@@ -48,7 +48,7 @@ export { globalAssert as assert, details, quote };
 // abbreviation 'q' for quote.
```

Comment:
This deprecation notice should be updated or dropped. I think the thing it's deprecating is now gone. The comment below seems like a recommendation rather than a deprecation.  isn't 'quote' gone?

Reply:
good catch